CHANGELOG.md

```diff
@@ -32,6 +32,10 @@
 * [#4477](https://github.com/bbatsov/rubocop/issues/4477): Warn when user configuration overrides other user configuration. ([@jonas054][])
 * [#5240](https://github.com/bbatsov/rubocop/pull/5240): Make `Style/StringHashKeys` to accepts environment variables. ([@pocke][])
 * [#5395](https://github.com/bbatsov/rubocop/pull/5395): Always exit 2 when specified configuration file does not exist. ([@pocke][])
+* [#4704](https://github.com/bbatsov/rubocop/issues/4704): Move `Lint/EndAlignment` to the `Layout` namespace. ([@bquorning][])
+* [#4704](https://github.com/bbatsov/rubocop/issues/4704): Move `Lint/EndAlignment` and `Lint/DefEndAlignment` to the `Layout` namespace. ([@bquorning][])
+* [#4704](https://github.com/bbatsov/rubocop/issues/4704): Move `Lint/EndAlignment`, `Lint/DefEndAlignment`, and `Lint/BlockAlignment` to the `Layout` namespace. ([@bquorning][])
+* [#4704](https://github.com/bbatsov/rubocop/issues/4704): Move `Lint/EndAlignment`, `Lint/DefEndAlignment`, `Lint/BlockAlignment`, and `Lint/ConditionPosition` to the `Layout` namespace. ([@bquorning][])
```

Comment:
Seems like there's some duplication of entries here. 🙂

Reply:
Fixed. Sorry.